packages/back-end/src/queryRunners/QueryRunner.ts

```diff
@@ -656,29 +652,31 @@ export abstract class QueryRunner<
     logger.debug("Creating query for: " + name);
     const concurrencyLimitReached = await this.concurrencyLimitReached();
     const dependenciesComplete = dependencies.length === 0;
-    const readyToRun =
-      dependenciesComplete && !runAtEnd && !concurrencyLimitReached;
+    const readyToRun = dependenciesComplete && !runAtEnd;
```

Comment:
Changed the definition of `readyToRun` to be slightly clearer. A query hitting the concurrency limit is "ready to run"; the reason it's not running is external.